backend/app/adapter/recaptcha/recaptcha_integration_test.go

```diff
@@ -47,6 +49,13 @@ func TestReCaptcha_Verify(t *testing.T) {
 				Hostname:      "s.time4hacks.com",
 			},
 		},
+		{
+			name: "unsuccessful request produced an error",
+			httpResponse: nil,
+			httpErr: errors.New("Service encountered an error"),
+			expRes: requester.VerifyResponse{},
+			expErr: errors.New("Error validating captcha response"),
```

Comment:
Let's use has error instead of comparing error message direct.

Reply:
Done